include/stylesheet.css

```diff
@@ -260,6 +261,40 @@ img.circle {
     cursor: pointer;
 }
 
+/* tab switcher tooltips */
+.circle:hover .tab_tooltip {
+    position:absolute;
+    top:22px;
+    left: -8px;
+    padding: 4px;
+    display: inline-block;
+    border-radius: 4px;
+    border: 1px solid var(--tertiary_color);
+    background-color: yellow;
+    color:black;
+}
+
+.tab_tooltip{
+display:none;
+}
+
+/* creo un quadratino, lo ruoto di 45° e lo posiziono */
+.tab_tooltip:before {
+    content: "";
+    background-color: yellow;
+    border-top: 1px solid var(--tertiary_color);;
+    border-left: 1px solid var(--tertiary_color);;
```

Comment:
need to adjust these to be border-bottom and border-right to accommodate the new positioning of the tooltip